### PR TITLE
Validate TGZ and multiversion variables

### PR DESCRIPTION
**Pull request by dokshina**
Multiversion and TGZ options are added to variables types schema.

### CODE REVIEW

File: library/cartridge_validate_config.py [hunk 1]
```diff
@@ -142,6 +143,16 @@ def validate_types(vars):
         'weight': int,
         'vshard_group': str,
         'cartridge_enable_tarantool_repo': bool,
+        'cartridge_install_dir': str,
+        'cartridge_instances_dir': str,
+        'cartridge_conf_dir': str,
+        'cartridge_run_dir': str,
+        'cartridge_data_dir': str,
```
Comment (hackallcode) [inline library/cartridge_validate_config.py:150]: Is it possible to check that it's not a simple string, but a path?

Comment (dokshina) [reply library/cartridge_validate_config.py:150]: I think that string is enough.

Comment (hackallcode) [reply library/cartridge_validate_config.py:150]: It seems to me that it is worth thinking about how to make a check for all paths in the config (not in this pull request).

**Review by hackallcode — DISMISSED** [review]

**Review by hackallcode — APPROVED** [review]
Great!

**Pull request merged**